config/serverless.yml

```diff
@@ -48,7 +48,7 @@ data.search.sessions.enabled: false
 advanced_settings.enabled: false
 
 # Enforce restring access to internal APIs see https://github.com/elastic/kibana/issues/151940
-# server.restrictInternalApis: true
+server.restrictInternalApis: true
```

Comment:
@jloleysens @pgayvallet suggests a CLI [implementation](https://github.com/elastic/kibana/pull/161733/files#r1262308667).

Reply:
I suggested to do something else in https://github.com/elastic/kibana/pull/161733/files#r1262308667 because the purpose of the linked PR was to enabled restricted API mode for tests, and enabling it globally wasn't the best way of doing so.

If your PR intends to enabled the restriction in production, then uncommenting the value in the serverless config file makes sense.